README.md

```diff
@@ -68,385 +47,34 @@ and the applications that run on top of it.
-            "send_back_addr": "/ip4/18.197.157.119/tcp/42962"
-          }
-        },
-        "knownAddresses": [
-          "/ip4/172.26.59.86/tcp/30333",
-          "/ip4/127.0.0.1/tcp/30333",
-          "/ip6/::1/tcp/30333",
-          "/ip4/18.197.157.119/tcp/30333",
-          "/ip4/10.0.0.2/tcp/30333",
-          "/ip4/10.0.1.18/tcp/30333"
-        ],
-        "latestPingTime": {
-          "nanos": 108101687,
-          "secs": 0
-        },
-        "open": true,
-      },
-      "QmP3zYRhAxxw4fDf6Vq5agM8AZt1m2nKpPAEDmyEHPK5go": {
-        "enabled": true,
-        "endpoint": {
-          "listening": {
-            "listen_addr": "/ip4/10.44.1.11/tcp/30101",
-            "send_back_addr": "/ip4/191.232.49.216/tcp/3010"
-          }
-        },
-        "knownAddresses": [
-          "/dns4/p2p.testnet-4.kusama.network/tcp/30100",
-          "/ip4/127.0.0.1/tcp/30100",
-          "/ip4/191.232.49.216/tcp/30100",
-          "/ip4/10.244.0.10/tcp/30100"
-        ],
-        "latestPingTime": {
-          "nanos": 717286051,
-          "secs": 0
-        },
-        "open": true,
-        "versionString": "parity-polkadot/v0.5.0-4e53ad1-x86_64-linux-gnu (unknown)"
-      },
-
-      [ ........ ]
-
-    },
-    "notConnectedPeers": {
-
-      [ secure validator ]
-
-      "QmZSocEssLWHYCY6mqR99DcSFEpMb95fVeMsScrY8jqBm8": {
-        "knownAddresses": [
-          "/ip4/10.0.0.1/tcp/30333",
-          "/ip4/10.0.1.18/tcp/30333",
-          "/ip4/10.0.1.152/tcp/30333",
-          "/ip4/147.75.199.231/tcp/30333"
-        ],
-        "latestPingTime": {
-          "nanos": 375552762,
-          "secs": 0
-        },
-      }
-
-      [ ........ ]
-
-    },
-    "peerset": {
+This is done through the ansible playbook and polkadot-validator role located at
+[ansible](/ansible), basically the role performs this actions:
 
-      [ all known nodes shown here, reported connected to all, secure validator with 0 reputation ]
+* Software firewall setup, for the validator we only allow the proxy, SSH and, if
+enabled, node-exporter ports.
+* Configure journald to tune log storage.
+* Create polkadot user and group.
+* Configure NGINX proxy
+* Setup polkadot service, including binary download.
+* Polkadot session management, create session keys if they are not present.
+* Setup node-exporter if the configuration includes it.
 
-      "QmP3zYRhAxxw4fDf6Vq5agM8AZt1m2nKpPAEDmyEHPK5go": {
-        "connected": true,
-        "reputation": 1115
-      },
-      "QmPjNcWNZjNrjVFzkNYR6jH7HLqyU7j9piczUyNoxce1fD": {
-        "connected": true,
-        "reputation": 3500
-      },
-      "QmZSocEssLWHYCY6mqR99DcSFEpMb95fVeMsScrY8jqBm8": {
-        "connected": true,
-        "reputation": 0
-      },
+# Note about upgrades from the sentries setup
 
-      [ ........ ]
+The current version of polkadot-secure-validator doesn't allow to create and configure
+sentry nodes. However, the terraform files and ansible roles of this latest version
```

Comment:
I'd reformulate like this: 
"Although the terraform files and ansible .... , the validator would be configured to work .... "

Reply:
Better that way, done